Sources/Extensions/SwiftStdlib/IntExtensions.swift

```diff
@@ -172,6 +172,12 @@ public extension Int {
 	}
 	// swiftlint:enable identifier_name
 
```

Comment:
Can you add documentation comments for that?

Reply:
Added doc comment.